port/freertos/golioth_sys_freertos.c

```diff
@@ -14,38 +14,45 @@
  * Time
  *------------------------------------------------*/
 
-uint64_t golioth_sys_now_ms() {
+uint64_t golioth_sys_now_ms()
```

Comment:
These are moving from the FreeRTOS port to the system ports (ESP-IDF, ModusToolBox, etc.) in https://github.com/golioth/golioth-firmware-sdk/pull/486, but I think it's fine to keep it here for now.